flink-dist/pom.xml

```diff
@@ -205,6 +205,32 @@ under the License.
 			</exclusions>
 		</dependency>
 
+		<!-- Default file system support -->
+
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-hadoop-fs</artifactId>
+			<version>${project.version}</version>
+			<exclusions>
+				<exclusion>
+					<groupId>org.apache.flink</groupId>
+					<artifactId>flink-shaded-hadoop2</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-mapr-fs</artifactId>
+			<version>${project.version}</version>
```

Comment:
This works because flink-shaded-hadoop2 is a transitive optional dependency? We don't exclude it explicitly here.